SalesforceFoundation/NPSP
# Gift Entry: use the single gift entry form from an account or contact record

# Critical Changes

# Changes
Enter a gift using the gift entry form directly from a contact or account record. 

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Instructions
- create a template with contact fields: first name, last name, email , account fields: name and website and opportunity fields: amount, closed date and stage
- add the New Gift button to the Contact layout
 - add the New Gift button to the Household layout
- go to a contact record and click the New Gift link
- check that the contact fields populate on the form
- fill in all the required fields
- save the gift, opportunity is created
- go to a household account and click New Gift
- check that the account fields populate on the form
- fill in all the required fields
 - save the gift, opportunity is created
